openmetadata-ui/src/main/resources/ui/src/pages/GlossaryPage/GlossaryPageV1.component.tsx

```diff
@@ -514,6 +530,9 @@ const GlossaryPageV1 = () => {
             }
           });
         });
+        if (selectedData?.name !== updatedData.name) {
+          handleRedirection(response.fullyQualifiedName);
+        }
       } else {
         throw jsonData['api-error-messages']['update-description-error'];
       }
```

Comment:
Is this the correct error message for the operation?

Reply:
hmm nice catch, seems like it is not!